template_src/config.xml

```diff
@@ -27,7 +27,6 @@
     </author>
     <content src="index.html" />
     <!-- Whitelist configuration. Refer to https://cordova.apache.org/docs/en/edge/guide_appdev_whitelist_index.md.html -->
-    <plugin name="cordova-plugin-whitelist" spec="1" />
```

Comment:
I think the comment above would no longer belong here. I wonder if we should consider putting this information into the generated README.md or something.

Reply:
I deliberately left that comment since the tags _below_ the `plugin` tag are the actual whitelist configuration. Compare to https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-whitelist/